src/lib/components/Cytoscape.react.js

```diff
@@ -270,6 +280,17 @@ class Cytoscape extends Component {
             refreshLayout();
         });
 
+        cy.on('dragfree', 'node', (_) => {
+            this.props.setProps({
+                elements: cy.elements('').map((item) => {
+                    return {
+                        data: item.json().data,
+                        position: item.json().position,
```

Comment:
Looks like `item.json().group` will tell you whether the item is one of the `'nodes'` or `'edges'` -- add an `if` statement to only include position if it's a node?

Reply:
This should be fixed now! 💃